photon-client/src/main/scala/com/linkedin/photon/ml/estimators/GameParams.scala

```diff
@@ -148,7 +153,7 @@ class GameParams extends FeatureParams with PalDBIndexMapParams with EvaluatorPa
   var modelOutputMode: ModelOutputMode = ALL
 
   /**
-   * Number of output files to write for each random effect model.
+   * Number of output files to write for each random effect model
```

Comment:
We better agree on this now or it'll thrash and drive me crazy:

Periods at the end of the body for method documentation, no periods for member and argument documentation. Deal?